devicon.json

```diff
@@ -8164,6 +8164,38 @@
         "color": "#41B883",
         "aliases": []
     },
+   {
```

Comment:
Whoopsie. v8 isn't inserted in the correct place alphabetically. Could you please move it so that `devicon.json` is alphabetically sorted?

Reply:
i  did it

Reply:
Nice, but now the indentation is wrong again 😅

Reply:
> Nice, but now the indentation is wrong again 😅

where ? please

Reply:
Altnames and aliases